### PR TITLE
Update ways of working

### PR DESCRIPTION
**Pull request by seadowg**
An update to our `README.md` and `CONTRIBUTING.md` to reflect the way we've been working for the last couple of releases. It felt like a good point to write this down for ourselves, and for any external contributors looking to get involved.

I think as part of this PR, we should also update our Transifex/strings workflow so it works better with release branching. Currently, we run into problems because we only get translations for `master` which means that the translations might not be appropriate even present for a different release branch.

### CODE REVIEW

**Comment by seadowg** [comment]
@ln @grzesiek2010 I'm going to have a think about the Transfix stuff referenced in the description, but it'd be great to get feedback on the changes here.

**Comment by lognaturel** [comment]
Looks good to me! The only thing that surprised me is that https://github.com/orgs/getodk/projects/9 is private. May I make it public?

**Review by lognaturel — APPROVED** [review]

**Comment by seadowg** [comment]
> Looks good to me! The only thing that surprised me is that https://github.com/orgs/getodk/projects/9 is private. May I make it public?

Absolutely! I'd actually meant to request that as a comment here and then forgot. I'm unable to change visibility for org projects 🤷‍♂️

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -81,15 +81,16 @@ We encourage everyone to participate in the review process to get familiar with
...
 
 ## Merging pull requests
 
-Pull requests can be merged by committers once they are accepted or once they have been accepted and have been labelled as `behavior verified` unless:
+Pull requests can be merged by maintainers once they are accepted or once they have been accepted and have been labelled as `behavior verified` unless:
 
-* There is a currently a "code freeze" due to an [ongoing release](/README.md#release-cycle)
 * [CI](https://app.circleci.com/pipelines/github/getodk/collect) is currently failing
 
+If a PR is being merged to a release branch rather than `master`, any strings added as part of the changes should also be added to `master` (with `tools:ignore="UnusedResources"`) as a follow-up PR so that they can be translated.
```
Comment (seadowg) [inline docs/CONTRIBUTING.md:92]: There's definitely smoother workflows, but they all require a high degree of non trivial automation so let's go with this for the moment.

File: README.md [hunk 2]
```diff
@@ -34,9 +34,12 @@ Please note that the `master` branch reflects ongoing development and is not pro
...
+Releases are planned to happen every 2-3 months (resulting in ~4 releases a year). Soon before (or just after) the end of one release cycle, the core team will plan a new set of work for the next release based on the [ODK Roadmap](https://getodk.org/roadmap), bugs and crashes identified in and previous releases and other required or preemptive maintenance. This work will be broken down into Github Issues (for things that aren't already) by [@seadowg](https://github.com/seadowg) and is then is added into Collect's prioritised [backlog](https://github.com/orgs/getodk/projects/9/views/8) for the core team (and any external contributors) to work on day to day. Sometimes issues will be assigned to core team members before they are actually started (moved to "in progress") to make it clear who's going to be working on what.
```
Comment (grzesiek2010) [inline README.md]: Double is in `...and is then is added into Collect's prioritised...`

File: docs/CONTRIBUTING.md [hunk 3]
```diff
@@ -29,7 +29,9 @@ To contribute code to ODK Collect, you will need to open a [pull request](https:
 
         git checkout -b NAME_OF_YOUR_BRANCH
 
-1. If there is an [issue](https://github.com/getodk/collect/issues) corresponding to what you will work on, leave a comment on issue asking to claim it. If there is no issue yet, create one to provide background on the problem you are solving.
+1. If there is an [issue](https://github.com/getodk/collect/issues) corresponding to what you will work on, leave a comment on issue asking to claim it. If there is no issue yet, create one to provide background on the problem you are solving. Issues that have been prioritised by the core team can be found in Collect's [backlog](https://github.com/orgs/getodk/projects/9/views/8).
```
Comment (grzesiek2010) [inline docs/CONTRIBUTING.md]: It should be on the issue in `leave a comment on issue asking to claim it`

File: docs/CONTRIBUTING.md [hunk 4]
```diff
@@ -29,7 +29,9 @@ To contribute code to ODK Collect, you will need to open a [pull request](https:
...
+
+1. If you are an external contributor, make sure to discuss working on the issue with a maintainer who will then assign you to it. If you're on the core team you can either assign yourself to an issue in [the backlog](https://github.com/orgs/getodk/projects/9/views/8) and move it to "in progress" or discuss adding an item to the backlog with the reset of the team.
```
Comment (grzesiek2010) [inline docs/CONTRIBUTING.md]: `rest of the team` not `reset of the team`

File: README.md [hunk 5]
```diff
@@ -34,9 +34,12 @@ Please note that the `master` branch reflects ongoing development and is not pro
...
+
+Fixes to a previous release should be merged to a "release" branch (`v2023.2.x` for example) so as to leave `master` available for the current releases work. If hotfix changes are needed in the current release as well then these can be merged in as a PR after hotfix releases (generally easiest as a single PR for the whole hotfix release). This approach can also be used if work for the next release starts before the current one is out - the next release continues on `master` while the release is on a release branch.
```
Comment (grzesiek2010) [inline README.md]: `for the current releases work` should this be plural? I know that it's possible that we work on more than one release on the same time like now for example be usually it should be one.

File: README.md (hunk 2 above)
Comment (grzesiek2010) [inline README.md]: `in and previous` should that be just `in previous`?

**Review by grzesiek2010 — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (seadowg) [reply README.md]: I think best to think of just one "current" release for `master`. As you say, what we're doing now is the exception.

Comment (grzesiek2010) [reply README.md]: So you agree it shouldn't be plural?

Comment (seadowg) [reply README.md]: Ah it should be `release's`. I've fixed it.

**Review by grzesiek2010 — APPROVED** [review]

**Pull request merged**